### DISCUSSION TITLE
Language in Batch endpoint

### DISCUSSION THREAD

**Discussion by devtronic** [etsy/open-api#1005] [Q&A]
We tried to request multiple listings using /listings/batch with a specific language but the language parameter is ignored.

Are there plans to support the ?language=de parameter in the batch endpoint?

https://github.com/etsy/open-api/discussions/868#discussioncomment-5451720

**Comment by DennisBillbee** [comment]
Is it possible to get an answer here? A lot of our customers are complaining about the wrong language in their details.

**Comment by Gareth064** [comment]
Etsy Devs don't monitor this GitHub for issues.

Email them at developer@etsy.com and you can tell them the problem you're having.

**Reply by DennisBillbee** [reply]
Thanks. I will contact them

**Reply by devtronic** [reply]
That's different to what the README says: 
![image](https://github.com/etsy/open-api/assets/2362734/fe0589e9-b8b4-4285-8cf3-157d8cbf6ea3)

**Reply by turboturtleD** [reply]
Does that change anything?
